KsApi/MockService.swift

```diff
@@ -1827,7 +1827,10 @@
 
       switch response {
       case let .success(pledgedProjectsData):
-        return Just(pledgedProjectsData).setFailureType(to: ErrorEnvelope.self).eraseToAnyPublisher()
+        return Just(pledgedProjectsData).setFailureType(to: ErrorEnvelope.self).delay(
+          for: 0.01,
+          scheduler: DispatchQueue.main
+        ).eraseToAnyPublisher()
```

Comment:
This is pretty cool. Testing Combine publishers isn't something I've been able to dive into, so it's cool to start seeing it in our app.